modules/programs/gh.nix

```diff
@@ -131,7 +134,7 @@ in {
     programs.git.extraConfig.credential = mkIf cfg.gitCredentialHelper.enable
       (builtins.listToAttrs (map (host:
         lib.nameValuePair host {
-          helper = "${cfg.package}/bin/gh auth git-credential";
+          helper = (if cfg.install then "${cfg.package}/bin/gh" else "gh") ++ "auth git-credential"
```

Comment:
I'm not sure what the best approach for a fallback would be here? Another option for the user to provide a path?